docs/usage.md

```diff
@@ -1,153 +1,306 @@
 
-- `test`
-  - A profile with a complete configuration for automated testing
-  - Includes links to test data so needs no other parameters
-- `docker`
-  - A generic configuration profile to be used with [Docker](https://docker.com/)
-- `singularity`
-  - A generic configuration profile to be used with [Singularity](https://sylabs.io/docs/)
-- `podman`
-  - A generic configuration profile to be used with [Podman](https://podman.io/)
-- `shifter`
-  - A generic configuration profile to be used with [Shifter](https://nersc.gitlab.io/development/shifter/how-to-use/)
-- `charliecloud`
-  - A generic configuration profile to be used with [Charliecloud](https://hpc.github.io/charliecloud/)
-- `conda`
-  - A generic configuration profile to be used with [Conda](https://conda.io/docs/). Please only use Conda as a last resort i.e. when it's not possible to run the pipeline with Docker, Singularity, Podman, Shifter or Charliecloud.
+| Mandatory                     | Optional                       |
+| ----------------------------- | ------------------------------ |
+| genome<sup>1</sup>            | gnomad_af<sup>4</sup>          |
+| vcfanno_resources<sup>2</sup> | reduced_penetrance<sup>5</sup> |
+| vcfanno_toml<sup>3</sup>      | vcfanno_lua                    |
+| vep_cache_version             | vep_filters                    |
+| vep_cache                     |                                |
 
-### `-resume`
+<sup>1</sup>Genome version is used by vep. You have the option to choose between GRCh37 and GRCh38.
+<sup>2</sup>Path to vcf files and their indices used by vcfanno. Sample file [here](https://github.com/ramprasadn/test-datasets/blob/raredisease/reference/vcfanno_resources.txt).
+<sup>3</sup>Path to a vcfanno configration file. Sample file [here](https://github.com/ramprasadn/test-datasets/blob/raredisease/reference/vcfanno_config.toml).
+<sup>4</sup>Gnomad vcf file can be downloaded from here https://gnomad.broadinstitute.org/downloads.
+<sup>5</sup>Used by genmod while ranking the variants. Sample file [here](https://github.com/ramprasadn/test-datasets/blob/raredisease/reference/reduced_penetrance.tsv).
 
-Specify this when restarting a pipeline. Nextflow will use cached results from any pipeline steps where the inputs are the same, continuing from where it got to previously. For input to be considered the same, not only the names must be identical but the files' contents as well. For more info about this parameter, see [this blog post](https://www.nextflow.io/blog/2019/demystifying-nextflow-resume.html).
+#### 7. SV annotation & Ranking
 
-You can also supply a run name to resume a specific run: `-resume [run-name]`. Use the `nextflow log` command to show previous run names.
+| Mandatory                  | Optional           |
+| -------------------------- | ------------------ |
+| genome                     | reduced_penetrance |
+| svdb_query_dbs<sup>1</sup> | score_config_sv    |
+| vep_cache_version          | vep_filters        |
```

Comment:
Should we add a sentence or example file for reduced penetrance, score_config and vep_filters?
Maybe also describe that vep_filters will split the results in a "clinical" list and research list.

Reply:
done